### ISSUE TITLE
Error when enabling a kinesis streaming destination for a dynamodb table

### ISSUE THREAD

**Issue by jbward1** [localstack/localstack#3600]
# Type of request: This is a ...

[X ] bug report
[ ] feature request

# Detailed description
When attempting to create a kinesis destination for dynamodb, i'm receiving An unknown operation was requested.

## Expected behavior
A successful response + localstack dynamo configured to start table data replication to the specified Kinesis data stream
```
{
   "DestinationStatus": "ACTIVE",
   "StreamArn": "arn:aws:kinesis:us-east-1:000000000000:stream/kinesis-dest-test-stream",
   "TableName": "kinesis-dest-test"
}
```

## Actual behavior
The following error is returned:
```
An error occurred (UnknownOperationException) when calling the EnableKinesisStreamingDestination operation: An unknown operation was requested.
```

# Steps to reproduce
1. Create Dynamo Table
2. Create Kinesis Stream
3. Enable Kinesis Streaming Destination 

## Command used to start LocalStack
SERVICES=dynamodb,kinesis localstack start

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
1. Create Dynamo Table
 ```
awslocal dynamodb create-table \
  --region us-east-1 \
  --table-name kinesis-dest-test \
  --attribute-definitions AttributeName=id,AttributeType=S \
  --key-schema AttributeName=id,KeyType=HASH \
  --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
```
2. Create Kinesis Stream
 ```awslocal kinesis create-stream \
  --stream-name kinesis-dest-test-stream \
  --shard-count 1
```
3. Enable Kinesis Streaming Destination 
```
awslocal dynamodb enable-kinesis-streaming-destination \
  --table-name kinesis-dest-test \
  --stream-arn arn:aws:kinesis:us-east-1:000000000000:stream/kinesis-dest-test-stream \
```

**Comment by stevenpila** [comment]
Hi, any updates on this one? we are also encountering the same problem.

**Comment by whummer** [comment]
Thanks for reporting @jbward1 . This should be fixed in #3889 - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists. Thanks!
